core/src/com/unciv/logic/map/MapUnit.kt

```diff
@@ -105,6 +105,9 @@ class MapUnit {
     var promotions = UnitPromotions()
     var due: Boolean = true
     var isTransported: Boolean = false
+    
+    var religionSpreadsUsed: Int = 0
```

Comment:
Generalization., think towards the future. 'UsesLeft', or something of the sort.